.buildkite/pipeline.build.yml

```diff
@@ -334,6 +334,58 @@
     - echo "--- Running the script with fault of networking delay"
     - ray job submit --address http://localhost:8265 --runtime-env python/ray/tests/chaos/runtime_env.yaml --working-dir python/ray/tests/chaos -- python potato_passer.py --num-actors=3 --pass-times=1000 --sleep-secs=0.01
 
+
+- label: ":kubernetes: :mending_heart: :runner: chaos network delay many job submissions"
+  conditions: ["RAY_CI_LINUX_WHEELS_AFFECTED"]
+  instance_size: medium
+  commands:
+    - |
+      cleanup() {
+        if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi
+        kind delete cluster
+      }
+      trap cleanup EXIT
```

Comment:
consider make all these steps into a bash script.

buildkite is not very good at concat-ing complicated shell commands into a script properly.

Reply:
will do a refactor after this - I plan to make it a matrix test: {workload} x {fault}